study-builder/studies/atcp/snippets/countries-with-states/picklist-question-American Samoa-state-province-required.conf

```diff
@@ -0,0 +1,105 @@
+{
+  "questionType": "PICKLIST",
+  "selectMode": "SINGLE",
+  "renderMode": "DROPDOWN",
+  "isRestricted": false,
+  "isDeprecated": false,
+  "additionalInfoHeaderTemplate": null,
+  "additionalInfoFooterTemplate": null,
+  "validations": [
+    {
+      "ruleType": "REQUIRED",
+      "allowSave": true,
+      "hintTemplate": {
+        "templateType": "TEXT",
+        "templateCode": null,
+        "templateText": "$STATE_PROVINCE_req_hint",
+        "variables": [
+          {
+            "name": "STATE_PROVINCE_req_hint",
+            "translations": [
+              { "language": "en", "text": ${i18n.en.registration_state_province_req_hint} },
+              { "language": "de", "text": ${i18n.de.registration_state_province_req_hint} },
+              { "language": "es", "text": ${i18n.es.registration_state_province_req_hint} },
+              { "language": "fr", "text": ${i18n.fr.registration_state_province_req_hint} },
+              { "language": "hi", "text": ${i18n.hi.registration_state_province_req_hint} },
+              { "language": "it", "text": ${i18n.it.registration_state_province_req_hint} },
+              { "language": "ja", "text": ${i18n.ja.registration_state_province_req_hint} },
+              { "language": "pl", "text": ${i18n.pl.registration_state_province_req_hint} },
+              { "language": "pt", "text": ${i18n.pt.registration_state_province_req_hint} },
+              { "language": "ru", "text": ${i18n.ru.registration_state_province_req_hint} },
+              { "language": "tr", "text": ${i18n.tr.registration_state_province_req_hint} },
+              { "language": "zh", "text": ${i18n.zh.registration_state_province_req_hint} }
+            ]
+          }
+        ]
+      }
+    }
+  ],
+  "picklistLabelTemplate": {
+    "templateType": "TEXT",
+    "templateCode": null,
+    "templateText": "$state_province_picklist_label",
+    "variables": [
+      {
+        "name": "state_province_picklist_label",
+        "translations": [
+          {
+            "language": "en",
+            "text": ${i18n.en.registration_choose_state_province}
+          }
+        ]
+      }
+    ]
+  },
+  "groups": [],
+  "picklistOptions": [
+    
+  ],
```

Comment:
Why are we asking user to answer a picklist question when there's no picklist options to choose from? This question is also marked as required? How is user supposed to answer?

Reply:
Removed unused file